D6StarWars/D6StarWars.css

```diff
@@ -8,11 +8,23 @@
     color:white;
 }
 
-.charsheet .sheet-version::after {
-    content: '2.34';
+.charsheet div.version-top{
+    position: absolute;
+    right: 5px;
+    top: 30px;
+}
+
+.charsheet div.logo{
+    display: inline-block;
+    position: absolute;
+    left: 150px;
```

Comment:
Every single block has been manually set. This results in block overflow and will fuck larger sheets.